lib/Doctrine/ORM/Query/AST/Functions/ConcatFunction.php

```diff
@@ -38,7 +38,7 @@ public function getSql(SqlWalker $sqlWalker)
             $args[] = $sqlWalker->walkStringPrimary($expression);
         }
 
-        return call_user_func_array([$platform, 'getConcatExpression'], $args);
+        return $platform->getConcatExpression(...$args);
```

Comment:
@Majkl578 @Ocramius Maybe something for our CS as well?

Reply:
@carusogabriel not really that important, plus `call_user_func_array()` is quite magic, so I'd avoid fiddling with it at CS level.